lib/webpack-cli.js

```diff
@@ -11,6 +11,11 @@ class WebpackCLI extends GroupHelper {
         this.groupMap = new Map();
         this.groups = [];
         this.processingErrors = [];
+        this.commandNames = {
```

Comment:
Main this could be called `defaultCommands`. What do you think?

Reply:
yeah, makes sense :+1: